### DISCUSSION TITLE
settings.json can not work

### DISCUSSION THREAD

**Discussion by GGGBB** [MoonRide303/Fooocus-MRE#72] [Q&A]
settings.json can not work,how to solve?

**Comment by barepixels** [comment]
check typo

**Comment by MoonRide303** [comment]
@GGGBB
1. Make sure you're using MRE, not vanilla.
2. Check if file name is settings.json (make sure to enable showing file extensions in Windows).

Simplest test is copying provided settings-example.json to settings.json (without any changes), and then checking if MRE starts in Advanced mode - if it does, then you can review the settings, and adjust it to your needs.
